src/types/reviewRequest.ts

```diff
@@ -12,7 +12,7 @@ export interface User {
   label: string
 }
 
-type FileType = "page" | "nav" | "setting" | "file" | "image"
+type FileType = "page" | "nav" | "setting" | "file" | "image" | "placeholder"
```

Comment:
add another type like `DisplayedFileType` that's just `Omit<FileType, "placeholder">` and use it for the display level stuff